sdk/digitaltwins/azure-digitaltwins-core/API design.md

```diff
@@ -1178,44 +1178,41 @@ Async APIs
     /**
      * Creates one or many models.
      * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return The list of created models
+     * @return A {@link PagedFlux} of created models and the http response.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public Mono<List<ModelData>> createModels(List<String> models) { }
-
-    /**
-     * Creates one or many models.
-     * @param models The list of models to create. Each string corresponds to exactly one model.
-     * @return A REST response containing the list of created models.
-     */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
-    public Mono<Response<List<ModelData>>> createModelsWithResponse(List<String> models) { }
+    public PagedFlux<ModelData> createModels(List<String> models) { }
```

Comment:
don't you need createmodels with response?

Reply:
Also why not Iterable here?

Reply:
This is the Async version, so it has to be PagedFlux
And we don't need the one with response since PagedFlux will have the response in there.